pennylane/fermi/fermionic.py

```diff
@@ -147,15 +147,16 @@ def __add__(self, other):
         method"""
 
         self_fs = FermiSentence({self: 1.0})
+        use_jax = qml.math.get_interface(other) == "jax"
```

Comment:
You can perhaps combine "jax" and "numpy" here and then simplify the if condition below and at other places.